frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,46 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
+  data: RunInstancePayload,
+): Promise<boolean> => {
```

Comment:
I highly doubt that this returns a boolean on success

Reply:
Updated to reflect json response of the success function.